contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeListItem/index.vue

```diff
@@ -30,7 +30,8 @@
       <VListTileContent
         class="description-col pa-2 grow"
         :class="{
-          'mt-1': !isCompact
+          'my-4': !isCompact,
+          'my-2': isCompact,
```

Comment:
These styling changes in this file seems to have broken I had created according to the design. What was the reasoning?